actionpack/test/dispatch/exception_wrapper_test.rb

```diff
@@ -53,6 +53,25 @@ def backtrace
       end
     end
 
+    if defined?(ErrorHighlight)
```

Comment:
More precisely, this condition also needs to see if error_highlight version is 0.4.0 or higher.

Reply:
I have fixed it, thank you!

(After the commit, I noticed that this might be covered by Rails' Gemfile because this is a test, but the change would be good to have.)